firefox-ios/Client/Utils/WebCacheUtils.swift

```diff
@@ -0,0 +1,69 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import WebKit
+
+final class WebCacheUtils {
```

Comment:
curious, are we trying to clear data similar to what we do in settings? Are we able to reuse the same code there instead of creating a new utility? If we do decide to go this route, can we add tests?

Reply:
That is right Cyndi, clear option from Settings may do exactly the same thing. Will check. Thank you.